DynamicActiveQuery.php

```diff
@@ -155,12 +202,95 @@ public function createCommand($db = null)
             % (`?) \{
                 ( [a-z_\x7f-\xff][a-z0-9_\x7f-\xff]* (?: \. [^.|\s]+)* )
                 (?:  \| (binary (?:\(\d+\))? | char (?:\(\d+\))? | time (?:\(\d+\))? | datetime (?:\(\d+\))? | date
-                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?(?:,\d\d?)?\))?
-                        | int(eger)? | (?:un)? signed (?:\int(eger)?)?)  )?
+                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?,\d\d?\))?
+                        | int(eger)? | (?:un)? signed (?:\s\int(eger)?)?)  )?
             \} \1 %ix
```

Comment:
yes, there needs to be one or more whitespace chars. so `\s` is ok. (should even be `\s+`). but that doesn't explain why you escape the `i` in `int`

is this what we need?

          | int(eger)? | (?:un)? signed (?:\s+int(eger)?)?)  )?